doc/makeConnected/pgr_makeConnected.rst

```diff
@@ -52,7 +52,7 @@ Signatures
 
     pgr_makeConnected(edges_sql)
 
-    RETURNS SET OF (seq, source, target, cost)
+    RETURNS SET OF (seq, node_from, node_to)
```

Comment:
Why are you doing this change?

Reply:
Because the function returns SET OF (seq, node_from, node_to) 
I can make it to return (seq, source, target)